Terminology changes and descriptions (#1048)
mattermost/mattermost-handbook: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md

# Terminology

The language we use at Mattermost is extremely important. When we use consistent terminology it means we're all speaking the same language, and our customers know what to expect whether they're on the website or in the product documentation.

Terminology standardization started in 2022. The project has a large scope and includes:

- Standardizing headings to make them sentence case.
- Standardizing all text in the System Console to make them sentence case.
- Standardizing how we refer to our products and plans, as well as the mechanisms available to use them.
- Changing the casing of words to lowercase where appropriate.

> - Making our Edition and License page consistent and clear.

Reviewer: May want to clarify that we're talking about the Edition and License page within the System Console

> - Making our Edition and License page consistent and clear.

Author: ```suggestion
- Making our Edition and License page in the System Console consistent and clear.
```

> - Creating a UI text library of terms and strings for the UX team to use when creating mocks. This can also be used cross-functionally and by external partners.

Reviewer: ```suggestion
- Creating a UI text library of terms and strings for teams and external partners to use when creating mockups, designs, wireframes, and integrations.
```

The [outline of the project is documented here](https://docs.google.com/document/d/1ImzS3UhCkFzMF2-6FNByw9W2r8Gvx1sudBA0FjazVH0/edit#heading=h.crjl4fz9pz1n).

## Sentence case headings

We decided to change our headings to sentence case to:

> * Decrease friction for contributors so they wouldn't have to "take a stab" at getting heading capitalization right.

Reviewer: "guess" might be another plain English term for the idiom "take a stab" :)

Author: That is very true, thank you!

> * Decrease friction for contributors so they wouldn't have to "take a stab" at getting heading capitalization right.

Author: ```suggestion
* Decrease friction for contributors so they wouldn't have to guess at getting heading capitalization right.
```

* Bring our documentation format in line with current industry standards around readability.
* Make our documentation easier to read.

> This is a work in progress with the documentation having been completed in Q1 of 2022. The scope of the project extends to the Mattermost website, developer documentation, Handbook, and integrations content. There is no date set for completion.

Reviewer: are the docs complete? maybe safer to say "...with documentation updates delivered in Q1 of 2022."

> This is a work in progress with the documentation having been completed in Q1 of 2022. The scope of the project extends to the Mattermost website, developer documentation, Handbook, and integrations content. There is no date set for completion.

Author: ```suggestion
This is a work in progress with documentation updates delivered in Q1 of 2022. The scope of the project extends to the Mattermost website, developer documentation, Handbook, and integrations content. There is no date set for completion.
```
